### ISSUE TITLE
Toolbox menu items overlap background 

### ISSUE THREAD

**Issue by microbit-mark** [microsoft/pxt-microbit#4265]
If your window is too small for the block it seems to allow for overlapping. This happens at about 50% width.

Reproduces in Chrome. 

![Screenshot2021_07_07_154945](https://user-images.githubusercontent.com/31242877/124791353-e3a55c00-df43-11eb-95de-b8171a04f568.jpg)
![9E7A4AA9-EADF-4958-A6E6-D8EFBEE85779](https://user-images.githubusercontent.com/31242877/124791358-e4d68900-df43-11eb-98e5-01e60dfd0a44.jpg)
![90D3AE9A-0308-4C7B-8D78-606C460DD64F](https://user-images.githubusercontent.com/31242877/124791360-e56f1f80-df43-11eb-9a62-32d8828eaaf6.jpg)

**Comment by abchatra** [comment]
@microbit-mark Is the issue here the block bleeds out?

**Comment by microbit-mark** [comment]
> @microbit-mark Is the issue here the block bleeds out?

Yes. Seems to be at 50% window size. We noticed it in the data log blocks, but you can also see it in Images and Math as an example
